src/SFA.DAS.Apprenticeships.Web/Infrastructure/EmployerClaims.cs

```diff
@@ -0,0 +1,11 @@
+using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.Apprenticeships.Web.Infrastructure
+{
+    [ExcludeFromCodeCoverage]
+    public static class EmployerClaims
+    {
+        public static string AccountsClaimsTypeIdentifier => "http://das/employer/identity/claims/associatedAccounts";
```

Comment:
Are we happy with these being http? sonarqube seems unhappy

Reply:
Yes, this is purely a claim key